sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -673,6 +673,12 @@ Exceptions:
 
 * Throws synchronously with error code `invalid` if the graph contains a
   node which targets a device not present in `syclContext`.
+  
+* Throws synchronously with error code `invalid` if the graph contains
+  any isolated node which is not reachable from all other nodes.
```

Comment:
It's not obvious to me that this needs to be an error, I think we can represent multiple root nodes. What aspect of this do you think will be problematic?

Reply:
I don't think DAGs required that all nodes have to be connected. A graph with two root nodes that aren't connected to each other is still a valid graph IMO, and there's no reason we couldn't support it. Disallowing it because user's might make a mistake and not define their edges properly seems unnecessarily limiting.